MaxText/generate_distillation_data.py

```diff
@@ -26,7 +26,7 @@
     --hf-access-token <access token> \
     --batch-size 1024 --num-batches 100 \
     --num-generations 2 \
-    --max-output-length 128 --max-target-length 256 \
+    --max-prefill-length 64 --max-target-length 2048 \
```

Comment:
`max-prefill-length=64` is very small, many prompts will be longer, could you have 256 or 512 in this example?